packages/sdk/src/wallet/node/wallets/hosted/turnkey/TurnkeyWallet.ts

```diff
@@ -84,7 +84,7 @@ export class TurnkeyWallet extends Wallet {
   }
 
   private async createAccount(): Promise<LocalAccount> {
```

Comment:
Is there a reason to keep this differentiation?

Reply:
oh yeah, it probably makes sense to rename this to `createSigner` as well